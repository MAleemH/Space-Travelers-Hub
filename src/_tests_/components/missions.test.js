import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import Missions from '../../components/missions/missions';
import store from '../../redux/store';

describe('Missions', () => {
  it('Renders correctly for reserved state', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Missions />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
