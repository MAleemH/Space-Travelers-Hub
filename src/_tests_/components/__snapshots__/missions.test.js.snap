// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions Renders correctly for reserved state 1`] = `
<div
  className="container"
>
  <table
    className="table-container"
  >
    <thead
      id="thead"
    >
      <tr>
        <th
          className="th missionSize"
        >
          Missions
        </th>
        <th
          className="th descrSize"
        >
          Description
        </th>
        <th
          className="th statusSize"
        >
          Status
        </th>
        <th
          className="th empty"
        >
           
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
